utils/datautils.js

```diff
@@ -48,6 +49,39 @@ function getClaimById(claimId) {
   return db.Claim.findById(claimId)
 }
 
+function getConflictedClaims(claim,issueUrlDetail) {
+  projectName = '/' + issueUrlDetail.project + '/'
+  issueId = ['/' + issueUrlDetail.id + '/', '/' + issueUrlDetail.id ]
+  return db.Claim.findAll({
+    where : {
+      [Op.and] : [
+        {
+          issueUrl: {
+            [Op.like]: '%' + projectName + '%'
+          }
+        },
+        {
+          [Op.or] : [
+            {
+              issueUrl: {
+                [Op.like]: '%' + issueId[0]
+              },
+              issueUrl: {
+                [Op.like]: '%' + issueId[1]
+              }
+            }
+          ]
+        },
```

Comment:
You can have one common pattern for this.

Reply:
okay, I will try

Reply:
> @hereisnaman please help me with semantic changes I did git pull, still, I am getting them. Also, review changes and See screenshots and gifs.